.github/workflows/integration_test.yml

```diff
@@ -21,11 +21,11 @@ jobs:
 
```

Comment:
While we are on the topic of upgrading the dependencies, can you update the Poetry version on line 4?
Versions `1.8.4` (released 11 days ago) or version `1.8.3` (released 6 months ago) are suitable.

https://pypi.org/project/poetry/#history

Reply:
The poetry version is not up-to-date in the other two github workflow files either